utils/udp_client.go

```diff
@@ -78,14 +79,19 @@ func NewAgentClientUDP(hostPort string, maxPacketSize int) (*AgentClientUDP, err
 
 // EmitZipkinBatch implements EmitZipkinBatch() of Agent interface
 func (a *AgentClientUDP) EmitZipkinBatch(spans []*zipkincore.Span) error {
```

Comment:
Yeah, but to remove it here I need to remove it from the thrift.idl and to do that I need to remove this from all the clients first. So just stubbing out for now.